### PR TITLE
Translation-plugin support and better startpage handling

### PR DESCRIPTION
**Pull request by micgro42**
This adds two improvements:
1. A config-option has been added that removes the translation-prefix before it assigns the feedback manager. This simplifies the configuration from
```
de:abc	foo@example.com
en:abc	foo@example.com
fr:abc	foo@example.com
fr:def	bar@example.com
```
to
```
abc	foo@example.com
def	bar@example.com
```

2. A config options has been added that includes the a page in the parent namespace if the page has the same name as the namespace

Also I've added a svg-icon for the admin menu.

### CODE REVIEW

File: admin.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg style="width:24px;height:24px" viewBox="0 0 24 24">
+    <path d="M9,22A1,1 0 0,1 8,21V18H4A2,2 0 0,1 2,16V4C2,2.89 2.9,2 4,2H20A2,2 0 0,1 22,4V16A2,2 0 0,1 20,18H13.9L10.2,21.71C10,21.9 9.75,22 9.5,22V22H9M16,14V13C16,11.67 13.33,11 12,11C10.67,11 8,11.67 8,13V14H16M12,6A2,2 0 0,0 10,8A2,2 0 0,0 12,10A2,2 0 0,0 14,8A2,2 0 0,0 12,6Z" />
+</svg>
```
Comment (splitbrain) [inline admin.svg:3]: could be optimized

File: action.php [hunk 2]
```diff
@@ -119,6 +119,18 @@ public function getFeedbackContact($id) {
         $conf = confToHash(DOKU_CONF . 'plugin_feedback.conf');
 
         $ns = $id;
+
+        if ($this->getConf('span_translations')) {
+            $ns = $this->adjustForTanslations($id);
+
+        }
```
Comment (splitbrain) [inline action.php]: weird empty line

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+# we need additional plugin(s) for testing
```
Comment (splitbrain) [inline requirements.txt:1]: this file should probably contain the translation plugin!?

File: _test/getFeedbackContact.test.php [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        $trans = plugin_load('helper', 'translation', true);
+
+        $actual_output = $action->getFeedbackContact($input);
+
+        $this->assertEquals($expected_output, $actual_output, $message);
+    }
+
+
+
+}
```
Comment (splitbrain) [inline _test/getFeedbackContact.test.php]: lots of blank lines here

**Review by splitbrain — APPROVED** [review]
Apart from some minor formatting issues and the missing requirement for the translation plugin this looks good.

File: requirements.txt (hunk 3 above)
Comment (micgro42) [reply requirements.txt:1]: Added in 1129556

**Pull request merged**